mod/groups/views/default/object/discussion_reply.php

```diff
@@ -35,22 +36,34 @@
 	));
 }
 
-$subtitle = "$poster_text $date";
-
 if (elgg_in_context('activity')) {
 	$content = '<div class="elgg-output elgg-inner" data-role="comment-text">';
 	$content .= elgg_view('output/text', array(
 		'value' => elgg_get_excerpt($reply->description)
 	));
 	$content .= '</div>';
+
+	$date_link = elgg_view('output/url', [
+		'href' => $reply->getURL(),
+		'text' => $date,
+		'is_trusted' => true,
+	]);
 } else {
 	$content = elgg_view('output/longtext', array(
 		'value' => $reply->description,
 		'class' => 'elgg-inner',
 		'data-role' => 'discussion-reply-text',
 	));
+
+	$date_link = elgg_view('output/url', [
+		'href' => "#elgg-object-{$reply->guid}",
```

Comment:
I figured this would preserve whatever querystring was used to reach the comment without the more costly getUrl(). If it's a deal breaker I'll change it